arbnode/espresso_caff_node.go

```diff
@@ -31,6 +31,7 @@ type EspressoCaffNodeConfig struct {
 	RetryTime               time.Duration           `koanf:"retry-time"`
 	HotshotPollingInterval  time.Duration           `koanf:"hotshot-polling-interval"`
 	HotshotPollingTimeout   time.Duration           `koanf:"hotshot-polling-timeout"`
+	ParseRetryLimit         int                     `koanf:"parse-retry-limit"`
```

Comment:
nit: can we change the naming? and make it more discrete on what are we parsing etc